### PR TITLE
Set up redundancy and resource levels on infrastructure

### PR DESCRIPTION
**Pull request by pezholio**
This allows us to explicitly set the number of instances we're running in a service plan. There is the option to do autoscaling, but this gives us redundancy if one instance was to fall over.

### CODE REVIEW

File: azure/resource_groups/app/template.json [hunk 1]
```diff
@@ -32,9 +32,33 @@
...
+    },
+    "appServicePlanTier": {
+      "type": "string",
+      "defaultValue": "Standard"
+    },
+    "appServicePlanSize": {
+      "type": "string",
+      "defaultValue": "1"
+    },
+    "vspAppServicePlanInstances": {
```
Comment (erbridge) [inline azure/resource_groups/app/template.json]: Can this move down to live alongside the other VSP app service plan parameters?

File: azure/resource_groups/app/template.json [hunk 2]
```diff
@@ -32,9 +32,33 @@
...
     "vspAppServiceDockerImage": {
       "type": "string"
     },
+    "vspAppServicePlanTier": {
+      "type": "string",
+      "defaultValue": "Standard"
+    },
+    "vspAppServicePlanSize": {
+      "type": "string",
+      "defaultValue": "2"
```
Comment (erbridge) [inline azure/resource_groups/app/template.json]: Is there a reason for this to have a different default from the web app service plan?

File: azure/resource_groups/app/template.json [hunk 3]
```diff
@@ -32,9 +32,33 @@
...
+      "type": "string",
+      "defaultValue": "Standard"
+    },
+    "appServicePlanSize": {
+      "type": "string",
+      "defaultValue": "1"
+    },
+    "vspAppServicePlanInstances": {
+      "type": "int",
+      "defaultValue": 1
```
Comment (erbridge) [inline azure/resource_groups/app/template.json]: Is there a reason for this to have a different default from the web app service plan?

Comment (erbridge) [inline azure/resource_groups/app/template.json]: This should also be set to 2 on `production` for the same reasons that the web app service will be.

File: azure/resource_groups/app/template.json (hunk 2 above)
Comment (pezholio) [reply azure/resource_groups/app/template.json]: Nah, I guess I kinda saw the VSP as an ancilliary service from the main app, but there's a good chance it'll get just as much traffic as the main app, so happy to change it and review usage as we get real traffic

**Pull request merged**